### PR TITLE
IFIX-561 : Junit COA

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: domain-services/ifix-master-data-service/src/test/java/org/egov/producer/ProducerTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.egov.producer;
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/test/java/org/egov/producer/ProducerTest.java:1]: Are we using kafka in the master data service? 
If not delete everything related to kafka from the project.

File: domain-services/ifix-master-data-service/src/test/java/org/egov/util/CoaUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        LinkedHashMap<String, List<Government>> listLinkedHashMap = new LinkedHashMap<>();
+        listLinkedHashMap.put("government", governmentCreateResponse.getGovernment());
+
+        doReturn(listLinkedHashMap).when(serviceRequestRepository).fetchResult((String) any(), (GovernmentSearchRequest) any());
+
+        List<Government> actualResult = coaUtil.searchTenants(requestHeader, chartOfAccount);
+
+        assertNotNull(actualResult);
+        assertTrue(actualResult.size() > 0);
+    }
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/test/java/org/egov/util/CoaUtilTest.java:68]: Create a new method for searchTenantInvalid which would throw error, the one where the returned result from the service request repository is empty.

File: domain-services/ifix-master-data-service/src/test/resources/application_test.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #server.servlet.contextPath=/ifix-master-data
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/test/resources/application_test.properties:1]: Remove unused properties

File: domain-services/ifix-master-data-service/src/test/java/org/egov/util/CoaUtilTest.java (hunk 2 above)
Comment (pintu-eGov) [reply domain-services/ifix-master-data-service/src/test/java/org/egov/util/CoaUtilTest.java:67]: done

File: domain-services/ifix-master-data-service/src/test/resources/application_test.properties (hunk 3 above)
Comment (pintu-eGov) [reply domain-services/ifix-master-data-service/src/test/resources/application_test.properties]: done

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**